js/webpack.config.js

```diff
@@ -1,11 +1,18 @@
 var webpack = require('webpack');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const path = require('path');
 
 
 module.exports = {
 	entry: {
 		'muil': [
-			'babel-polyfill',
+			'@babel/polyfill',
 			__dirname + '/src/index.js',
+			path.join(
+				__dirname,
+				'..',
+				'sass/muil.sass',
+			)
```

Comment:
Rád to změním, ale jde to proti tomuhle:

As a lot of answers have pointed out, using path module is probably the best way. However, most of the solutions here have gone back to using slashes like:

path.join(__dirname+'../test/karma.conf.js')

However, by doing this, you're beating the purpose of using path. One uses path to do operations irrespective of the underlying OS (Linux, Windows etc). Just to give a bit of insight, you can do the path operations directly as string operations (like __dirname + '../test/karma.conf.js'. You do not do this because Linux uses forward slashes ( / ), Windows uses backward slashes ( \ ). This makes your application prone to errors when you port it across operating systems.

https://stackoverflow.com/questions/30845416/how-to-go-back-1-folder-level-with-dirname

Reply:
tak v tom případě `			path.join(__dirname, '..', 'sass', 'muil.sass')` aby to mělo smysl? A teda všude?